### ISSUE TITLE
Put new lines in gateway json responses

### ISSUE THREAD

**Issue by wfernandes** [cloudfoundry/log-cache#43]
Otherwise it messes up the terminal prompt.

```
pivotal@iliff warren-jason-log-cache $ curl http://127.0.0.1:8081/v1/meta
{"meta":{"warren":{"count":"48","oldest_timestamp":"1522973978408119744","newest_timestamp":"1522975260840841206"}}}pivotal@iliff warren-jason-log-cache $ wat?
```

**Comment by thepeterstone** [comment]
This appears to be an issue in grpc-gateway runtime and/or generated code. Feel free to dive in. We'll pull that fix in once it hits grpc-gateway.

**Comment by hdub2** [comment]
Fixed in 84f8901d96648413b5a47f74e256fded7dc3dc86 - expected to be incorporated in `log-cache-release` 1.4.8. Docker images should be available now.
